packages/celotool/src/lib/vm-testnet-utils.ts

```diff
@@ -0,0 +1,158 @@
+import {
+  AccountType,
+  generateGenesisFromEnv,
+  generatePrivateKey,
+  privateKeyToAddress,
+  privateKeyToPublicKey,
+} from '@celo/celotool/src/lib/generate_utils'
+import {
+  applyTerraformModule,
+  destroyTerraformModule,
+  initTerraformModule,
+  planTerraformModule,
+  TerraformVars,
+} from '@celo/celotool/src/lib/terraform'
+import {
+  uploadFileToGoogleStorage,
+  uploadGenesisBlockToGoogleStorage,
+} from '@celo/celotool/src/lib/testnet-utils'
+import { confirmAction, envVar, fetchEnv } from '@celo/celotool/src/lib/utils'
+import { writeFileSync } from 'fs'
+
+const secretsBucketName = 'celo-testnet-secrets'
+const terraformModule = 'testnet'
+
+// NOTE(trevor): The keys correspond to the variable names that Terraform expects
```

Comment:
nit: I think this is a useful comment to everyone, so feel free to take out the `NOTE()`